### PR TITLE
feat: some cleanup work

### PR DESCRIPTION
**Pull request by mbukeRepo**

### CODE REVIEW

File: meme-generator/src/components/Meme.jsx [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+  const clearInput = () => {
     setMeme((prevMeme) => ({
       ...prevMeme,
       topText: "",
       bottomText: "",
     }));
-  }
+  };
 
   const node = document.getElementById("meme-img");
```
Comment (hirwaaldo1) [inline meme-generator/src/components/Meme.jsx:48]: here it will be good if you use hooks like `useRef `or `useState `instead of using getElementById

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]
Everything looks good, but if you make the suggested change, it will be even better.

File: notes-app/src/components/Avatarbar.js [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
-    return (
-        <div>Avatarbar</div>
-    )
-}
+  return <div>Avatarbar</div>;
+};
 
-export default Avatarbar
\ No newline at end of file
+export default Avatarbar;
```
Comment (IbrahimBagalwa) [inline notes-app/src/components/Avatarbar.js]: What is the usecase of this component ?

File: notes-app/src/App.js [hunk 3]
```diff
@@ -51,7 +51,6 @@ const App = () => {
       }) || notes[0]
     );
   }, [currentNoteId, notes]);
-
   return (
     <div className="App dark">
```
Comment (IbrahimBagalwa) [inline notes-app/src/App.js:55]: the text `you have no notes` you can change the color because here is white

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
Well done Prince, few changes request

**Review by hirwaaldo1 — APPROVED** [review]
LGTM

**Review by IbrahimBagalwa — APPROVED** [review]
LGTM 👍

**Review by Ntwali-Josue — APPROVED** [review]

**Pull request merged**